src/components/MovieCard/MovieCard.js

```diff
@@ -4,11 +4,13 @@ import { Link } from 'react-router-dom';
 const MovieCard = ({ id, title, backdrop, averageRating }) => {
   return (
     <article className="movie-card">
-    <Link to={`/movies/${id}`}>
-      <img src={backdrop} alt={"image for " + title}/>
-    </Link>
-      <h1>{title}</h1>
-      <h2>Average Rating: {averageRating}</h2>
+      <Link to={`/movies/${id}`}>
+        <img src={backdrop} alt={"image for " + title}/>
+        <div className="movie-info-container">
+        <h1>{title}</h1>
+        <h2>Average Rating: {averageRating}</h2>
+      </div>
+      </Link>
     </article>
```

Comment:
It looks like you have a stray div tag here on line 12. Pretty minor though. Other than that it looks good, nice job!